tests/auth_tests/client.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from django.contrib.auth.views import INTERNAL_RESET_TOKEN
+from django.test import Client
+
+
+def extract_token_from_url(url):
+    token_search = re.search(r'/reset/.*/(.+?)/', url)
+    if token_search:
+        return token_search.group(1)
+
+
+class PasswordResetConfirmClient(Client):
```

Comment:
I didn't look at the tests yet when I wrote that, so if there are already tests covering that, it should be fine.